spark/examples/building-spark-for-emr.md

```diff
@@ -0,0 +1,61 @@
+Building a Spark Distribution for EMR
+=====================
+
+# DISCLAIMER
+
+The following information is for sharing with community on how to compile/build a custom installation of [Spark](http://spark.apache.org) for the EMR environment.   It does not nesecarilly represent how a "current" build of Spark may be done and provided by AWS or affliates.   Please use at your own risk.
```

Comment:
Thanks for documenting.

Spelling of "necessarily" and "affiliates" wrong.

Also, it would be useful to know what modifications AWS makes from the core source.  Otherwise, we can't be sure what the cause of differences might be.